app.rb

```diff
@@ -24,19 +26,6 @@ def run
     puts 'Thank you for using the Console Library App. Goodbye!'
   end
 
-  private
-
-  def display_options
-    puts "\nPlease choose an option:"
-    puts '1. List all books'
-    puts '2. List all people'
-    puts '3. Create a person'
-    puts '4. Create a book'
-    puts '5. Create a rental'
-    puts '6. List rentals for a person'
-    puts '7. Quit'
-  end
-
   def handle_choice(choice)
     case choice
     when 1 then list_all_books
```

Comment:
- [ ] Kindly consider moving this logic to main.rb file as main.rb file's role is to start the code and ask user for the options or to quit. app.rb should not have this logic.
- [ ] kindly consider moving `run, handle_choice` methods into main.rb.
requirement:
Make sure that the role of main.rb is only to start your code and ask the user for the initial choice of options (or to quit).